.github/workflows/main.yml

```diff
@@ -0,0 +1,41 @@
+name: build
+
+on: push
+
+jobs:
+  convert_via_pandoc_and_deploy:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Find and Replace
+        uses: jacobtomlinson/gha-find-replace@master
+        with:
+          find: "\n"
+          replace: " "
+          include: "index.txt"
+      - uses: pCYSl5EDgo/cat@master
+        id: index
+        with:
+          path: index.txt
+      - uses: docker://mahoyen/mypandoc:0.4.0
+        with:
+          args: >-
+            --from markdown+footnotes
+            --to latex
+            --template main.tex
+            --data-dir /github/workspace
+            --lua-filter settings/pandoc-gls.lua
+            --filter pandoc-citeproc
+            --bibliography=bibliography.bib
+            --out build/report.pdf
+            ${{ steps.index.outputs.text }}
```

Comment:
I would like to swap out this with `cat index.txt`, but it does not run the command then.